### DISCUSSION TITLE
how to slove css path error

### DISCUSSION THREAD

**Discussion by XiaBei-cy** [vuepress/core#1524] [Q&A] [answered]
I deployed the document in  amy/doc , the base: "/my/doc/" has been set in the configuration file.
But the back-to-top-icon css style url is ./assets/img/back-to-top.bdc03142.svg.
This is an error, causing the button image returning to the top to not display correctly. 
The correct one should be /amy/doc/assets/img/back-to-top.bdc03142.svg or ../img/back-to-top.bdc03142.svg. 
How to solve it?

![image](https://github.com/vuepress/core/assets/76262556/15fd5c16-3f45-4382-abc2-412c5bc0a850)

![image](https://github.com/vuepress/core/assets/76262556/d2c0a509-11a6-4ba5-b8c2-3486ade37eb9)

**Comment by Mister-Hope** [comment, marked as answer]
https://vuepress-theme-hope.github.io/v2/ is working fine, so you need to provide a reproduction.

**Reply by XiaBei-cy** [reply]
After restarting WebStorm and rebuilding, everything returned to normal, tkysm.
